Makefile.dep

```diff
@@ -12,6 +12,31 @@ ifneq (,$(filter nhdp,$(USEMODULE)))
   USEMODULE += oonf_rfc5444
 endif
 
+ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
+  USEMODULE += gnrc_netif
+  USEMODULE += netif_default
```

Comment:
Yes! This is why it should not be removed, but the dependency of `gnrc_netif_default` in `boards/*/Makefile.dep` should be! :-)